### PR TITLE
BrownBalance 2

### PR DESCRIPTION
**Pull request by BrownDairy**
Stirrings Edition
(Also added new cards w00p)

### CODE REVIEW

File: cardlibrary/ancientstirrings.lua [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+		["Id"] = 1237896695,
 		["Name"] = "Korblox Skeleton",
 		["Health"] = 300,
 		["Power"] = 100,
 		["Rarity"] = "Uncommon",
 		["AttackEffect"] = "Dash",
 		["Color"] = "Blue", 
 		["Archetype"] = "Skeleton",
-		["Cost"] = {["Blue"] = 1,},
+		["Cost"] = {["Blue"] = 2,},
```
Comment (Blitzwolfer) [inline cardlibrary/ancientstirrings.lua]: The majority has decided to give Serhat can't counterattack (counterblock). Increasing the cost by ONE ICON barely fixes the issue with Serhat/Korblox Skeleton.

Comment (Blitzwolfer) [reply cardlibrary/ancientstirrings.lua]: There's also the possibility of making Korblox Skeleton a token.

File: cardlibrary/ancientstirrings.lua [hunk 2]
```diff
@@ -477,9 +477,9 @@ local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Green"] = 3,},
 		["Effect"] = {
 			Name = "Pyrotechnics",
-			Description = "When targeted, transform into an Overseer Beast.",
+			Description = "When targeted, give all fighters 150 health and power, all Overseer fighters 150 health and power, and transform it into an Overseer Eye.",
```
Comment (Blitzwolfer) [inline cardlibrary/ancientstirrings.lua]: Rewrite the effect desc. as "When targeted, give all fighters 150 Health and Power, give all Overseers an additional 150 Health and Power, and transform this card into an Eye of Overseer.

File: cardlibrary/ancientstirrings.lua [hunk 3]
```diff
@@ -3219,7 +3220,124 @@ local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 			Target = "Self",
 		},
 		["Bio"] = "If I am mad, it is mercy! May the gods pity the man who in his callousness can remain sane to the hideous end!",
-	},	
+	},
```
Comment (Blitzwolfer) [inline cardlibrary/ancientstirrings.lua]: You sure Myrmiredon is okay with this?

File: cardlibrary/ancientstirrings.lua [hunk 4]
```diff
@@ -918,17 +918,17 @@ local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 	["Infected Mushrooms"] = {
 		["Id"] = 574315060,
 		["Name"] = "Infected Mushrooms",
-		["Health"] = 675,
-		["Power"] = 375,
+		["Health"] = 550,
```
Comment (Blitzwolfer) [inline cardlibrary/ancientstirrings.lua]: Bit weary about this change; post in card-changes.

File: cardlibrary/ancientstirrings.lua (hunk 1 above)
Comment (BrownDairy) [reply cardlibrary/ancientstirrings.lua]: Making it a token does not solve the problem, it rather halts it. any korblox skeletons still in decks will stay there, but nobody else can put one in their decks

**Pull request merged**